qiskit/test/mock/fake_qasm_simulator.py

```diff
@@ -34,7 +34,11 @@ def __init__(self):
             open_pulse=False,
             memory=True,
             max_shots=65536,
-            gates=[GateConfig(name='TODO', parameters=[], qasm_def='TODO')]
+            gates=[GateConfig(name='TODO', parameters=[], qasm_def='TODO')],
+            meas_kernels=['boxcar'],
+            discriminators=['linear', 'quadratic'],
+            default_meas_kernel='boxcar',
+            default_discriminator='linear'
```

Comment:
```suggestion
            default_discriminator='linear_discriminator'
```

Reply:
Addressed in https://github.com/Qiskit/qiskit-terra/pull/5131/commits/2bb40181110801931b8c77e7e114be2cfeeeca9f.